README.md

```diff
@@ -141,3 +141,28 @@ It's also possible to run the script without using docker. To this end, issue th
     mkdir $ROS_WS                    # and create it
 
     .moveit_ci/travis.sh
+
+## Running Locally For Testing in `OS X`
+
+First clone the repo you want to test:
```

Comment:
Instead of repeating all instructions here, just highlight the difference to a standard Linux build:
`export TRAVIS_OS_NAME=osx`